odysseus/HeraclesQuest.ts

```diff
@@ -253,8 +253,7 @@ export type HeraclesQuestReward = HeraclesQuestElement & ({
 export type HeraclesQuest = {
     display?: {
         icon?: HeraclesQuestIcon;
-
-        icon_background?: 'heracles:textures/gui/quest_backgrounds/default.png' | 'heracles:textures/gui/quest_backgrounds/circles.png';
+        icon_background?: string;
```

Comment:
Should be ResourceLocation.

Reply:
I'm confused... Odysseus is just changing one text file (FTBQ snbt's) to another (Heracles' json's). This isn't Minecraft mod code.

Reply:
Fixed with 1e52e4a